LayoutTests/http/tests/resources/network-simulator.py

```diff
@@ -146,6 +179,15 @@ def handle_log_resource_request(path):
     sys.stdout.write('Content-Type: text/html\r\n\r\n')
 
 
+if initial_delay_arg is not None:
+    initial_delay = int(initial_delay_arg)
+
+if chunk_size_arg is not None:
+    chunk_size = int(chunk_size_arg)
```

Comment:
We may want to ensure `chunk_size != 0`, otherwise, file.read(0) is valid and will return an empty string per https://docs.python.org/3.8/library/io.html#io.RawIOBase.read

Reply:
I'll check for `chunk_size != 0` before using